docs/source/reference/evaluations.rst

```diff
@@ -13,6 +13,15 @@ eval_pck
 ~~~~~~~~
 .. autofunction:: eval_pck
 
-eval_semantic_segmentation
-~~~~~~~~~~~~~~~~~~~~~~~~~~
-.. autofunction:: eval_semantic_segmentation
+
+eval_semantic_segmentation_iou
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. autofunction:: eval_semantic_segmentation_iou
+
+calc_semantic_segmentation_confusion
```

Comment:
This is not an alphabetical order. I guess you put these functions here because they are helper functions of `eval_semantic_segmentation_iou`. How about nesting these items to show the hierarchy more explicitly?